:(
```

  After user setup
    ✓ note (4980ms)
    1) open note form with hotkey


  11 passing (60s)
  1 failing

  1) After user setup
       open note form with hotkey:
     AssertionError: Timed out retrying after 4000ms: Expected to find element: `[data-cy-post-form-text]`, but never found it.
      at Context.eval (webpack://misskey/./cypress/e2e/basic.cy.ts:180:0)




  (Results)

  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Tests:        12                                                                               │
  │ Passing:      11                                                                               │
  │ Failing:      1                                                                                │
  │ Pending:      0                                                                                │
  │ Skipped:      0                                                                                │
  │ Screenshots:  1                                                                                │
  │ Video:        false                                                                            │
  │ Duration:     59 seconds                                                                       │
  │ Spec Ran:     basic.cy.ts                                                                      │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
```

--- a/cypress/e2e/basic.cy.ts
+++ b/cypress/e2e/basic.cy.ts
@@ -228,19 +228,6 @@ describe('After user setup', () => {
 
 		cy.contains('Hello, Misskey!');
   });
-
-	it('open note form with hotkey', () => {
-		// Wait until the page loads
-		cy.get('[data-cy-open-post-form]').should('be.visible');
-		// Use trigger() to give different `code` to test if hotkeys also work on non-QWERTY keyboards.
-		cy.document().trigger("keydown", { eventConstructor: 'KeyboardEvent', key: "n", code: "KeyL" });
-		// See if the form is opened
-		cy.get('[data-cy-post-form-text]').should('be.visible');
-		// Close it
-		cy.focused().trigger("keydown", { eventConstructor: 'KeyboardEvent', key: "Escape", code: "Escape" });
-		// See if the form is closed
-		cy.get('[data-cy-post-form-text]').should('not.be.visible');
-  });
 });
 
 // TODO: 投稿フォームの公開範囲指定のテスト

